### DISCUSSION TITLE
Different results with same sample symmetry

### DISCUSSION THREAD

**Discussion by AdityaVuppala** [mtex-toolbox/mtex#1013] [Ask Anything]
Hi,
I am currently using the Mtex version 5-4
I am trying to estimate the odf from a discrete sample. I am not concerted with the halfwidth paramater for now,

But, I have two cases where 
`CS = crystalSymmetry('432');
SS = `specimenSymmetry('mmm');

ori_n=loadOrientation('fullOrientations.dat', 'ColumnNames',{'Euler 1','Euler 2','Euler 3'}, CS, SS);
odf_n=calcODF(ori_n,'halfwidth', 5.5*degree);``

and in the second case where I assign the sample symmetry later
``CS = crystalSymmetry('432');
SS = `specimenSymmetry('1');
ss = `specimenSymmetry('mmm');

ori_n=loadOrientation('fullOrientations.dat', 'ColumnNames',{'Euler 1','Euler 2','Euler 3'}, CS, SS);
odf_n=calcODF(ori_n,'halfwidth', 5.5*degree);`
odf_n.SS = ss``

Actually both these cases give a different result.

Which of the approaches is correct?

Also, I have tried this example with an older verion Mtex4-5. This older version gives completely different results for the above 2 cases. Which version of MTEX do you prefer?

![Final_postOpti_1](https://user-images.githubusercontent.com/27729943/117312740-96463900-ae85-11eb-83ce-03485debd3b0.png)
![Final_postOpti_2](https://user-images.githubusercontent.com/27729943/117312744-97776600-ae85-11eb-964e-ea6ce73f0d29.png)

**Comment by kilir** [comment]
Hi,
please try the latest release version 5.7.

Cheers,
Rüdiger

**Reply by AdityaVuppala** [reply]
Dear Rüdiger,

I tried it also with the latest version of MTEX. But still I have the same issue.

The problem is, I first set the sample symmetry to "1" and then calculated the odf with calcDensity. Then changed the sample symmetry to "mmm"

Then I tried to start with the sample symmetry of "mmm" when calculating the odf with calcDensity. 

Both of them yield a different ODF. Which method is more reliable?

I have attached the ODF's from both the methods.

Thanks,
Aditya.
![Final_postOpti_1](https://user-images.githubusercontent.com/27729943/117324639-428d1d00-ae90-11eb-9e09-0b37f2cc6c2a.png)
![Final_postOpti_2](https://user-images.githubusercontent.com/27729943/117324642-43be4a00-ae90-11eb-90bc-53f4ce52f6e7.png)

**Reply by kilir** [reply]
Hi,
please have a look here, https://github.com/mtex-toolbox/mtex/discussions/770
Cheers,
Rüdiger
